pennylane/vqe/vqe.py

```diff
@@ -73,54 +79,58 @@ class Hamiltonian:
     :doc:`/introduction/chemistry` module can be used to generate a molecular
     Hamiltonian.
 
-    .. Warning::
-
-        Hamiltonians can be constructed using Pythonic arithmetic operations. For example:
+    If the coefficients are not trainable tensors, Hamiltonians can be
+    constructed using Pythonic arithmetic operations. For example:
```

Comment:
I tried, but of course this arithmetic does not work with tensors! I don't think we can support this easily. We could offer "to_torch" functions etc that convert the parameters!